src/content/docs/infrastructure/microsoft-azure-integrations/get-started/azure-integration-metrics.mdx

```diff
@@ -25,6 +25,12 @@ If the resulting metric name does not start with `azure.` after these transforma
 
 For more information about supported Azure resource types, see the [Azure Monitor Metrics documentation website](https://learn.microsoft.com/en-us/azure/azure-monitor/reference/supported-metrics/metrics-index).
 
+### Metric name matches one of the dimension
+
+When the metric name and the name of one of the dimensions are exactly the same, the dimension name is uncapitalized so both values are different: 
```

Comment:
```suggestion
Can you clarify what we're trying to say here? Are we recommending that customers lowercase the dimension name so that it's different than the metric name? Or is this happening automatically somehow, where the dimension name will be lowercased for them?
```

Reply:
@morecar Thanks for providing the context that these are automatic settings. Let's keep your original wording!